components/Comment/CommentDrawer.tsx

```diff
@@ -23,6 +23,7 @@ const CommentDrawer = ({ children, isOpen = false, handleClose }: Args) => {
   return (
     <div onClick={(e) => e.stopPropagation()}>
       <SwipeableDrawer
+        sx={{ zIndex: 1000000000 }}
```

Comment:
Unfortunately the reason for the ridiculously high z-index is due to other competing modals/elements with higher z-index